hack/update-deps.sh

```diff
@@ -68,4 +68,4 @@ if [[ -d hack/patches ]];then
     done
 fi
 
-update_licenses third_party/
+#update_licenses third_party/
```

Comment:
There's an issue going on with updating license. Not sure whether it's my setup only or others also.

Reply:
I think it's ok, `tektoncd/pipeline` removed `third_party` we can probably remove that as well.